int64 counter
int8 PRBS
float64[21] setPoint
